config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/mechanics/:id", to: "mechanics#show"
+  patch "mechanics/:id/add_ride", to: "mechanics#add_ride"
```

Comment:
This route is not restful. You could have done a post to "mechanics/:id/rides" and routed it to the "mechanic_rides#create" controller action. There are other options as well, but `add_ride` as a uri is not restful.

Reply:
Or, because of how you wrote your controller action, you could have done `patch "mechanics/:id", to: "mechanics#update"`, since you're just doing `mechanice.rides << ride` in your controller action so you're just adjusting the mechanic's data.